neuromation/api/url_utils.py

```diff
@@ -73,6 +74,7 @@ def _normalize_uri(uri: URL, username: str) -> URL:
         if path or uri.host:
             uri = uri.with_path(path)
 
+    assert uri.host, f"Could not resolve resource owner for URI {resource}"
```

Comment:
AFAIK, `uri.host is None` is impossible. Though, I could not find fully covering unit tests on `_normalize_uri` -- should I create an issue?